integrated_channels/api/v1/moodle/serializers.py

```diff
@@ -1,6 +1,8 @@
 """
```

Comment:
As discussed in the meeting, let's first test the serializer changes first by hitting the endpoint and verifying results. We should also check where else is this serializer being used.

Reply:
@sameenfatima78 for now unencrypted values should not be removed from the serializer?

Reply:
First we should check where exactly is this data being used, then we can merge the PR as is with encrypted columns.